app/models/keyword.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class Keyword < ApplicationRecord
```

Comment:
We can add model level validation
```suggestion
class Keyword < ApplicationRecord

validates: name, presence: true
```

Reply:
Thanks @junan :)
I've read that Rails won't raise an error if a keyword is bigger than 255 chars (and the string would simply be truncated). Is it a common practice to add also a max length validate? :)

Reply:
Okay, I've added:
```rubby
validates :name, presence: true, length: { maximum: 255 }
```

as well as added model tests to secure these 2x validation rules :) 
@junan  I keep the conversation open to hear from the previous comment :) Thanks!

Reply:
> I've read that Rails won't raise an error if a keyword is bigger than 255 chars (and the string would simply be truncated). Is it a common practice to add also a max length validate? :)

It is up to you I think, ` length: { maximum: 255 }` one better addition I  also think.I usually use `string` column for shorter  text and goes for `text`(it can store longeer text) column for longer text